boards/posix/native_posix/native_posix_64_defconfig

```diff
@@ -2,5 +2,6 @@
 
 CONFIG_SOC_POSIX=y
 CONFIG_BOARD_NATIVE_POSIX_64BIT=y
+CONFIG_SERIAL=y
```

Comment:
Don't do this. Turning the UART by default in native_posix is not ok, and would break things for many people (it already breaks sanitycheck).